### ISSUE TITLE
standard_layout body_attributes block BC break

### ISSUE THREAD

**Issue by andrewmy** [sonata-project/SonataAdminBundle#5256]
#### Sonata packages

```
sonata-project/admin-bundle              3.40.1 3.40.1 The missing Symfony Admin Generator
sonata-project/block-bundle              3.12.1 3.12.1 Symfony SonataBlockBundle
sonata-project/cache                     2.0.1  2.0.1  Cache library
sonata-project/core-bundle               3.11.2 3.11.2 Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.3.1  2.3.1  Symfony SonataDatagridBundle
sonata-project/doctrine-orm-admin-bundle 3.6.2  3.6.2  Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  1.9.1  1.9.1  Lightweight Exporter library
```

#### Symfony packages

Irrelevant.

#### PHP version

Irrelevant.

## Subject

A recent commit https://github.com/sonata-project/SonataAdminBundle/commit/c32414c7d64d1277851b04f8860ef13a05f4993a#diff-072497236e2956a1ca2fc388be05034eR104 introduced a required body data attribute. If you had overridden this block, your JS would be broken.

Moreover, the implementation had a malformed JSON object (strings instead of booleans), fixed in a next PR #5255.

## Steps to reproduce

- Install a previous release of Sonata
- Create your layout which extends standard_layout and override the body_attributes block

## Expected results

- All the JS page works

## Actual results

- data.config in Admin.read_config() is null

## Proposed solution

An upgrade note or move these data attributes to a new block.

**Comment by kunicmarko20** [comment]
cc @core23

**Comment by Webonaute** [comment]
Those 2 ticket is also related to this #5261 #5264

**Comment by andrewmy** [comment]
Taking the liberty to fix this in #5265

**Comment by psyray** [comment]
Issue already reported #5261 
PR already submitted and approved #5262 
Please check before opening issue
